### DISCUSSION TITLE
Serving other clients

### DISCUSSION THREAD

**Discussion by pedrosimao** [epicweb-dev/epic-stack#46] [General]
Full-stack apps (without APIs) are great and so much faster to develop, but in many cases we might have a second client, like a mobile app.
What would be the ideal approach to use Epic Stack to serve this second external client? Would it be possible to easily serve a GraphQL API if needed?

**Comment by kentcdodds** [comment]
I've written up docs about this: https://github.com/epicweb-dev/epic-stack/blob/main/docs/apis.md

**Reply by iakshay** [reply]
This works in practice. 

Curious on your thoughts on RESTful API semantics. It seems harder to build RESTful API in the current structure.

**Reply by kentcdodds** [reply]
What's hard about it?
